source/coding-the-legislation/legislation_parameters.md

```diff
@@ -95,43 +95,55 @@ public_sector:
      ...
 ```
 
-#### Create Scales
+#### Scales
 
-Scales are constituted of brackets. Brackets are defined by amounts, bases, rates, average rates and thresholds.
+Scales are complex parameters constituted of brackets. They offer convenient built-in tools for recurring calculation patterns.
 
-Sample `parameters/tax_on_salary/tax_scale.yaml`:
+For instance, a marginal rate test scale can be defined in a YAML file with:
+
+File `parameters/tax_on_salary/tax_scale.yaml`
 ```yaml
 description: Scale for tax on salaries
 brackets:
-- rate:
-    1950-01-01:
-      value: 0.0
-    2010-01-01:
-      value: 0.02
-  threshold:
-    1950-01-01:
-      value: 0.0
-- rate:
-    1950-01-01:
-      value: 0.2
-  threshold:
-    1950-01-01:
-      value: 2000
+  - rate: # Define a 1st bracket
+      1950-01-01:
+        value: 0.0  # The rate applied to the first bracket was 0% from 1950 to 2009
+      2010-01-01:
+        value: 0.02  # The rate applied to the first bracket has changed to 2% in 2010
+    threshold:
+      1950-01-01:
+        value: 0.0
+  - rate: # Define a 2nd bracket
+      1950-01-01:
+        value: 0.2  # The rate applied to the second bracket has been 20% since 1950
+    threshold:
+      1950-01-01:
+        value: 2000  # The 2nd bracket starts for wages beyond 2000
+metadata:
```

Comment:
The metadata should also contain the scale type, per #811 - admittedly this is not entirely consistent, as this `type` field is only used currently if the value is `single_amount`, but we can similarly document the values `marginal_amount`, `marginal_rate`, `linear_average`, i.e. document first and implement later ;)

Reply:
👌 
It doesn't feel great to me that we are using a _metadata_ property to alter core's behavior, but I'll just add that to my list of complaints about the scales 📝